ompi/mpi/c/Makefile.am

```diff
@@ -58,442 +497,25 @@ ompi_HEADERS = $(headers)
-        file_write_at.c \
-        file_write.c \
-        file_write_ordered_begin.c \
-        file_write_ordered.c \
-        file_write_ordered_end.c \
-        file_write_shared.c \
-        finalize.c \
-        finalized.c \
-        free_mem.c \
-        gather.c \
-        igather.c \
-        gather_init.c \
-        gatherv.c \
-        igatherv.c \
-        gatherv_init.c \
-        get_address.c \
-        get_count.c \
-        get_elements.c \
-        get_elements_x.c \
-        get_accumulate.c \
-        get_library_version.c \
-        get_processor_name.c \
-        get_version.c \
-        graph_create.c \
-        graph_get.c \
-        graph_map.c \
-        graph_neighbors_count.c \
-        graph_neighbors.c \
-        graphdims_get.c \
-        grequest_complete.c \
-        grequest_start.c \
-        group_c2f.c \
-        group_compare.c \
-        group_difference.c \
-        group_excl.c \
-        group_f2c.c \
-        group_free.c \
-        group_from_session_pset.c \
-        group_incl.c \
-        group_intersection.c \
-        group_range_excl.c \
-        group_range_incl.c \
-        group_rank.c \
-        group_size.c \
-        group_translate_ranks.c \
-        group_union.c \
-        ibsend.c \
-        improbe.c \
-        imrecv.c \
-        info_c2f.c \
-        info_create.c \
-        info_create_env.c \
-        info_delete.c \
-        info_dup.c \
-        info_f2c.c \
-        info_free.c \
-        info_get.c \
-        info_get_nkeys.c \
-        info_get_nthkey.c \
-        info_get_string.c \
-        info_get_valuelen.c \
-        info_set.c \
-        init.c \
-        init_thread.c \
-        initialized.c \
-        intercomm_create.c \
-        intercomm_create_from_groups.c \
-        intercomm_merge.c \
-        iprobe.c \
-        irecv.c \
-        irsend.c \
-        is_thread_main.c \
-        isend.c \
-        isendrecv.c \
-        isendrecv_replace.c \
-        issend.c \
-        lookup_name.c \
-        message_f2c.c \
-        message_c2f.c \
-        mprobe.c \
-        mrecv.c \
-        neighbor_allgather.c \
-        ineighbor_allgather.c \
-        neighbor_allgather_init.c \
-        neighbor_allgatherv.c \
-        ineighbor_allgatherv.c \
-        neighbor_allgatherv_init.c \
-        neighbor_alltoall.c \
-        ineighbor_alltoall.c \
-        neighbor_alltoall_init.c \
-        neighbor_alltoallv.c \
-        ineighbor_alltoallv.c \
-        neighbor_alltoallv_init.c \
-        neighbor_alltoallw.c \
-        ineighbor_alltoallw.c \
-        neighbor_alltoallw_init.c \
-        keyval_create.c \
-        keyval_free.c \
-        op_c2f.c \
-        op_commutative.c \
-        op_create.c \
-        op_f2c.c \
-        op_free.c \
-        open_port.c \
-        pack_external.c \
-        pack_external_size.c \
-        pack.c \
-        pack_size.c \
-        parrived.c \
+#interface_profile_sources = $(patsubst %.c.in,%.c,$(prototype_sources))
```

Comment:
Nit: Should remove this commented-out patsubst line (I agree that we don't want to use patsubst -- that's a GNU Make extension and won't work on BSD Make).